src/shapes/sprite.js

```diff
@@ -14,16 +14,18 @@ function SpriteShape(params) {
     dx: params.x,
     dy: params.y,
     speed: properties.speed,
-    animation: null
+    update: null,
+    animation: properties.animation === true ? true : false,
```

Comment:
But if `properties.animation` is equal `string` ? It return `true`, but isn't `true`.